Readme.md

```diff
@@ -58,36 +58,3 @@ The following packages are published from this repository:
 Nightly Builds are available on our MyGet feed:
 `https://www.myget.org/F/applicationinsights-dotnet-nightly/api/v3/index.json`
 These builds come from the develop branch. These are not signed and are not intended for production workloads.
```

Comment:
Instead of removing the whole section, we could say the 'milestones' in github could be used to find the upcoming release or something similar.